holidays/countries/brazil.py

```diff
@@ -198,9 +198,16 @@ def _populate(self, year):
             self[date(year, OCT, 5)] = "Criação de Roraima"
 
         if self.subdiv == "SC":
-            self[date(year, AUG, 11)] = (
-                "Criação da capitania," " separando-se de SP"
-            )
+            day_of_the_week = date(year, AUG, 11).weekday()
+            if (year <= 2017) or (day_of_the_week >= 5):
+                self[date(year, AUG, 11)] = (
+                    "Criação da capitania," " separando-se de SP"
+                )
+            else:
+                delta = 6 - day_of_the_week
+                self[date(year, AUG, 11 + delta)] = (
+                    "Criação da capitania," " separando-se de SP"
+                )
```

Comment:
```suggestion
            dt = date(year, AUG, 11)
            if year >= 2018:
                dt += rd(weekday=SU(+1))
            self[dt] = "Criação da capitania, separando-se de SP"
```
and add SU to import:
`from dateutil.relativedelta import TU, SU`